frontend/src/metabase/actions/components/ActionViz/Action.unit.spec.tsx

```diff
@@ -271,7 +271,7 @@ describe("Actions > ActionViz > Action", () => {
     it("should render as a button if no parameters are missing", async () => {
       await setup({
         settings: formSettings,
-        parameterValues: { "dash-param-1": "foo", "dash-param-2": "bar" },
+        parameterValues: { "dash-param-1": "foo", "dash-param-2": 2 },
```

Comment:
Action's `parameter_2` is [actually numeric](https://github.com/metabase/metabase/blob/978abfe65110ff788f447ec3ee90ba1dcc989234/frontend/src/metabase/actions/components/ActionViz/Action.unit.spec.tsx#L47), so the test started failing once we consolidated type checks and validation